.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
should we add this file to the `.gitignore` ?

perhaps if there is an EOS studio project file we can include it in the repo.

Reply:
Adding to .gitignore now...
I didn't use an EOS Studio project. just the regular way